7_child_vaccination.do

```diff
@@ -1,83 +1,215 @@
-
 ******************************
 *** Child vaccination ********
 ******************************   
-
 *c_measles	child			Child received measles1/MMR1 vaccination
-        gen c_measles  =. 
-		replace c_measles = 1 if (h9 ==1 | h9 ==2 | h9 ==3)  
-     	replace c_measles = 0 if h9 ==0  				
-	
+		gen c_measles  = inlist(h9,1,2,3)   if !inlist(h9,8,9,.)  
+
+		
+		gen c_polio0 = inlist(h0,1,2,3)  if !inlist(h0,8,9,.) 

```

Comment:
Can you put polio under the polio section? ( for example under *c_polio0)

Reply:
sure